pkg/rpcserver/rpcserver.go

```diff
@@ -156,20 +169,28 @@ func newImpl(ctx context.Context, cfg *Config, mgr Manager) (*Server, error) {
 			statNoExecDuration:     queue.StatNoExecDuration,
 		},
 	}
-	s, err := flatrpc.ListenAndServe(cfg.RPC, serv.handleConn)
+	return serv, nil
+}
+
+func (serv *server) Close() error {
+	return serv.serv.Close()
+}
+
+func (serv *server) Start() error {
```

Comment:
It's mostly for the unit test where we want to assert rpcserver's `server` creation without starting the actual RPC server.
In future, it will be useful if we would like to pass syz-manager's Manager rpcserver `server` object during Manager creation and start RPC server when all Manager's dependencies are set and ready to start.

Reply:
Maybe rename it to `Listen()`? That's what it's actually about.